### DISCUSSION TITLE
There is duplication in the results presented

### DISCUSSION THREAD

**Discussion by rlquxi** [dbeaver/dbeaver#22243] [Q&A]
Mac computers use dbeaver. Use the same SQL. The query result set shows duplicates. There is no problem with the source data and the statement. I've tried installing a different version and still get this issue.
<img width="608" alt="11" src="https://github.com/dbeaver/dbeaver/assets/154737635/a50e0dd8-e6ed-4b8f-b822-bb33b26a13e4">
<img width="599" alt="22" src="https://github.com/dbeaver/dbeaver/assets/154737635/8ab28597-fa50-4bdc-b1ac-e21952321705">

**Comment by LonwoLonwo** [comment]
Hello @rlquxi 

What database and driver version do you use?

**Reply by rlquxi** [reply]
Several databases, such as oracle and impala, have this problem, and dbeaver has tried it from versions 22 to 23

**Reply by rlquxi** [reply]
<img width="758" alt="image" src="https://github.com/dbeaver/dbeaver/assets/154737635/2e8911e2-4a0b-4160-a672-a4d754277330">

**Reply by rlquxi** [reply]
After each sql execution, you also need to manually click Refresh to display the correct results

**Reply by LonwoLonwo** [reply]
Could you please turn on these four settings and try again?

![2023-12-25 12_28_07-Preferences](https://github.com/dbeaver/dbeaver/assets/45152336/ae680be4-4166-441f-be7e-cb3b8eae6f4d)

**Reply by rlquxi** [reply]
After testing, after selecting these four options, no duplicate display problem was found, thank you
<img width="811" alt="image" src="https://github.com/dbeaver/dbeaver/assets/154737635/024e9a96-a504-4f2a-875c-659d1b2b6d1f">
